scripts/zip.js

```diff
@@ -3,9 +3,52 @@
 
   function getData() {
     $.getJSON('/data/manhattan.json', function(data) {
-      // Start here  :-)
-  }
-
+      // Run map to create a new array of returned objects with only the desired properties.
+      var modifiedNeighborhoods = data.features.map(function(object) {
+        return {
+          zip: object.properties.zip,
+          neighborhood: object.properties.neighborhood,
+          address: object.properties.address,
+          // Since the coordinates are in lng/lat order, and we need them in lat/lng order, reverse the array
+          // we get back by checking object.geometry.coordinates.
+          coordinates: object.geometry.coordinates.reverse()
+        };
+      });
+      console.log(modifiedNeighborhoods);
+      // From the modifiedNeighborhoods array, generate a new array of only the neighborhood names, reduced
+      // to only the unique neighborhood names.
+      var topFive = modifiedNeighborhoods.map(function(object) {
+        return object.neighborhood;
+      }).reduce(function(acc, next) {
+        if (acc.indexOf(next) === -1) {
+          acc.push(next);
+        }
```

Comment:
Good! Try to implement the above with the filter method (replacing reduce) for an additional exercise.